btr-web/btr-common-components/utils/debounce.ts

```diff
@@ -0,0 +1,16 @@
+export function debounce (func: Function, timeout = 250) {
```

Comment:
I would prefer we use something like https://vueuse.org/shared/useDebounceFn for these sorts of things. Thoughts?

Reply:
I like it!
I will remove this file, agreed it would be better, thanks for suggestion !